docs/managed-datahub/operator-guide/setting-up-remote-ingestion-executor.md

```diff
@@ -13,7 +13,7 @@ import FeatureAvailability from '@site/src/components/FeatureAvailability';
 
 :::note
 
-The Remote Executor can now also be used for monitoring of ingestion sources.
+The Remote Executor is only available for DataHub Cloud. If you are looking to use Observability features in Acryl DataHub Cloud, remote executor must be enabled in your Acryl instance. Please contact Acryl support to enable Remote Executor in your instance.
```

Comment:
This statement is not true right? 

If you want observability, you can ALSO use a local executor. You don't have to use a remote executor.

Reply:
It's either local or remote, we often use the term remote executor interchangeably whether or not we assume actual remote executor. But if you have a better wording, feel free to adjust as necessary.